<?xml version="1.0"?>

<launch>
	<arg name="file_name" default="path" />

	<node name="record_path_node" pkg="nuogeng" type="record_path_node" output= "screen">
		<param name="file_path" value="$(find nuogeng)/data/"/>
		<param name="file_name" value="$(arg file_name).txt"/>
		<param name="sample_distance" value="0.1" />
		<param name="utm_topic" value="/ll2utm_odom" />
	</node>
	
	<node name="nuogeng_node" pkg="nuogeng" type="nuogeng_node" output="screen">
		<param name="port_name" value="/dev/ttyUSB1"/>
		<param name="is_ll2utm" value="true"/>
		<param name="odom_topic" value="/ll2utm_odom"/>
	</node>

</launch>
